game-service/src/main/java/org/libertybikes/game/core/GameRound.java

```diff
@@ -237,5 +270,13 @@ public void startGame() {
                 e.printStackTrace();
             }
         }
+        state = State.RUNNING;
+    }
+
+    /**
+     * @return
+     */
+    public JsonObject toJson() {
```

Comment:
this method can be removed, since going from POJO->JSON will be handled automatically by JSONB